<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_PopupDial_IndicationChoice</class>
 <widget class="QWidget" name="C_PopupDial_IndicationChoice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <widget class="QCheckBox" name="checkBox_IndicationCIM10">
     <property name="whatsThis">
      <string>Evaluer les indications codées avec le CIM10</string>
     </property>
     <property name="text">
      <string>CIM 10</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="checkBox_IndicationBDM">
     <property name="whatsThis">
      <string>Evaluer les indications spécifiques à l'implémentation commerciale des données</string>
     </property>
     <property name="text">
      <string>Spécifique à la base</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkBox_SMR">
     <property name="toolTip">
      <string>Evaluer le SMR (service médical rendu)</string>
     </property>
     <property name="whatsThis">
      <string>Evaluer le SMR (service médical rendu)</string>
     </property>
     <property name="text">
      <string>SMR</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="checkBox_ASMR">
     <property name="whatsThis">
      <string>Evaluer l'ASMR (l'ajout de service médical rendu)</string>
     </property>
     <property name="text">
      <string>ASMR</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTreeWidget" name="treeWidget_Indications">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Indication</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ASMR / SMR</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Code</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
